htdocs/core/modules/import/import_xlsx.modules.php

```diff
@@ -116,24 +116,24 @@ public function __construct($db, $datatoimport)
 		require_once DOL_DOCUMENT_ROOT . '/includes/Psr/autoloader.php';
 		require_once PHPEXCELNEW_PATH . 'Spreadsheet.php';
 		$this->workbook = new Spreadsheet();
-		
```

Comment:
You should setup your IDE to "autoremove spaces at end of line" to avoid such errors (automatically fixed by "Sticker tool")